lib/src/helpers/events/providers.dart

```diff
@@ -5,12 +5,12 @@
 import 'dart:js_interop';
 import 'dart:js_interop_unsafe';
 
-import '../../../helpers.dart' show Device;
 import '../../../web.dart';
+import '../enums.dart' show Device;
 import 'streams.dart';
 
 /// Statically accessible `EventStreamProvider`s for all event types.
-class EventStreamProviders {
+sealed class EventStreamProviders {
```

Comment:
Could also make this `abstract sealed`.

Reply:
`abstract sealed` just adds the ability to extend...which...why?

https://dart.dev/language/modifier-reference

Reply:
Turns out `abstract sealed` doesn't even exist. :) `abstract final` it is then.